lib/TPP/Dialect/Tpp/TppUtils.cpp

```diff
@@ -175,9 +174,8 @@ static bool isTppUnaryOp(linalg::GenericOp linalgOp) {
 }
 
 // Returns true if the linalg.generic maps to a tpp.gemm.
+// TODO: we may want to remove now.
```

Comment:
Yes, this looks like it could just call the `linalgx` function directly.

Reply:
I know that in the past we didn't run `isTppOp` for matmul (don't remember why anymore) but shouldn't this be included here?

Reply:
Here the history: 
In the past we used not to check memref abstraction when mapping to TPP. Then we fix with: https://github.com/plaidml/tpp-mlir/commit/09815cc477df600caf70b331cfdc3cf9ce773fc9 and today since we want to handle tensor too we "revert" back. Does this answer your question? I assume at some point `isTPPMatmul` will re-surface back if we want to handle beta = 0. Now tpp.matmul = linalg.matmul but this will change soon.